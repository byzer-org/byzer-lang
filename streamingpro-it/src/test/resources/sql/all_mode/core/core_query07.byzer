--%comparator=tech.mlsql.it.IgnoreResultComparator

set t1 = '''
    { "id": 1, "name": "carl" }
    { "id": 2, "name": "paul" }
    { "id": 3, "name": "jack" }
    { "id": 4, "name": "rose" }
    { "id": 5, "name": "helen" }
''';

set t2 = '''
    { "id": 1, "age": 18 }
    { "id": 2, "age": 20 }
    { "id": 3, "age": 50 }
    { "id": 4, "age": 23 }
    { "id": 5, "age": 43 }
''';

load jsonStr.`t1` as t1;
load jsonStr.`t2` as t2;

select
    t1.id as id,
    t1.name as name,
    t2.age as age
from t1
join t2 on t1.id = t2.id
as t3;

select count(1) as account from t3 as t4;

!assert t4 ''':account == 5''' "Joined failure!";