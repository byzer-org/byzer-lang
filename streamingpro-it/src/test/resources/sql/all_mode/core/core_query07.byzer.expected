id,name,age
1,carl,18
2,paul,20
3,jack,50
4,rose,23
5,helen,43